### PR TITLE
WFCORE-5537 Improve error messages in LayersTest

### PR DESCRIPTION
**Pull request by parsharma**
Issue: https://issues.redhat.com/browse/WFCORE-5537

### CODE REVIEW

File: testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java [hunk 1]
```diff
@@ -105,6 +105,7 @@ public static void test(String root, Set<String> unreferenced, Set<String> unuse
                 invalidUnref.add(unref);
             }
         }
+        // If error an occurs please have a look at "unreferenced" java doc
         if (!invalidUnref.isEmpty()) {
             throw new Exception("Some unreferenced modules are un-expected " + invalidUnref);
```
Comment (bstansberry) [inline testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java:110]: This should still provide some context, e.g. the path to the 'test-standalone-reference' and 'test-all-layers' installations that were compared, e.g.

"Some unreferenced modules in installation <path to test-all-layers> are unexpected compared to <path to test-standalone-reference> -- "

**Review by bstansberry — CHANGES_REQUESTED** [review]
The "ERROR, some modules have not been provisioned: " message should also be improved.

File: testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java (hunk 1 above)
Comment (bstansberry) [reply testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java:113]: @parsharma I meant in the exception message.

Comment (bstansberry) [reply testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java:113]: @parsharma I meant in the exception message.

**Comment by parsharma** [comment]
Thanks, @bstansberry.

File: testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java [hunk 2]
```diff
@@ -105,8 +105,11 @@ public static void test(String root, Set<String> unreferenced, Set<String> unuse
                 invalidUnref.add(unref);
             }
         }
+
+        // If an ERROR occurs please refer to unreferenced java doc for more details
         if (!invalidUnref.isEmpty()) {
-            throw new Exception("Some unreferenced modules are un-expected " + invalidUnref);
+            throw new Exception("!!ERROR, some modules have not been provisioned-- " +
+                    " Some unreferenced modules in installation [path to test-all-layers] are unexpected compared to [path to test-standalone-reference] " + invalidUnref);
```
Comment (bstansberry) [inline testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java:112]: Those should be the actually paths.

**Pull request closed without merging**